### PR TITLE
Updates the link to point to AGCO's landing page

### PR DESCRIPTION
**Pull request by bltavares**
screen shot 2016-04-28 at 2 13 22 pm

### CODE REVIEW

**Comment by waldemarnt** [comment]
👍

**Comment by jeffhsta** [comment]
:+1:

File: source/index.html.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ language_tabs:
   - shell
 
 toc_footers:
-  - <a href='#'>Sign Up for a Developer Key</a>
+  - <a href='https://agco-fuse.github.io/'>Get in touch for a Developer Key</a>
```
Comment (glauco) [inline source/index.html.md]: What do you think about "Get in touch for Sandbox access"

Comment (bltavares) [inline source/index.html.md]: I like it.

**Pull request merged**